group_project_v2/templates/html/stages/team_evaluation.html

```diff
@@ -1,5 +1,5 @@
-<form class="review peer_review" data-review-type="peer_review" action="submit_review" method="POST">
+<div class="review peer_review" data-review-type="peer_review" data-action="submit_review" data-method="POST">
```

Comment:
@Kelketek LMS appends some staff-only content in a `section` tag. And section is not allowed inside a form, or, at least, it terminates form right after itself in Chrome. So we can't have a form with multiple XBlocks in it. When we get to making it a wider use we'll have to address a lot more serious concerns (i.e. API, pluggable services, XBlock as a course tab, etc.)